src/components/arcgis-wrapper/arcgis-wrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import { Map as ArcGisMap } from "@esri/react-arcgis";
```

Comment:
Make a basic test for the new file

Reply:
added new test file: arcgis-wrapper.spec.tsx